app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -51,7 +51,8 @@ def storage_resources
         :ems_ref          => resource.uuid,
         :logical_free     => resource.logical_free,
         :logical_total    => resource.logical_total,
-        :physical_storage => persister.physical_storages.lazy_find(resource.storage_system)
+        :physical_storage => persister.physical_storages.lazy_find(resource.storage_system),
+        :capabilities     => resource.capability_values_json
```

Comment:
Since we have a jsonb column why not parse this into an easily consumable hash?

Reply:
parse it here with JSON.parse ?

Reply:
I think we need to confirm this with @ereztheodorou , 
this field might be use for other things.

Reply:
@agrare ok. 
so we will parse those column here..